EC_finetune/__main__.py

```diff
@@ -289,7 +327,7 @@ def main():
         args.csv_headers = CAPTIONING_CSV_HEADERS
     else:
         args.csv_headers = EC_CSV_HEADERS
-        if args.language_model_lambda or args.weight_drift_lambda: 
+        if args.language_model_lambda or args.weight_drift_lambda:
```

Comment:
This feels weird; we could have an EC with neither of these right (i.e. with both lambdas at 0)? and so would still want to keep track of comm loss